tests/em/fdem/forward/test_FDEM_analytics.py

```diff
@@ -61,8 +61,8 @@ def setUp(self):
         prb.pair(survey)
 
         try:
-            from pymatsolver import PardisoSolver
-            prb.Solver = PardisoSolver
+            from pymatsolver import Pardiso
```

Comment:
This is a little tricky, there is a lot of code floating around that will try to import `PardisoSolver` and if it fails, fall back to `SolverLU`, so we might want to allow both for a bit to catch these?